packages/react-instantsearch-dom/src/lib/infiniteHitsCache/index.js

```diff
@@ -0,0 +1,3 @@
+export {
+  default as getInfiniteHitsSessionStorageCache,
```

Comment:
Any better name? 😟

Reply:
Since the function instantiate our cache object I would have gone with the `create` prefix instead of `get`: `createInfiniteHitsSessionStorageCache`

Reply:
@yannickcr I thought the same at first until I realize that it doesn't create it, but _retrieve_ it. The `create` prefix gets confusing when you see the function getting called at each render, because it doesn't actually create the cache:

```jsx
function Hits() {
  return (
    <InfiniteHits
      hitComponent={Product}
      cache={createInfiniteHitsSessionStorageCache()}
    />
  );
}
```

Reply:
For the SessionStorage it does not create the cache itself (since it is handled by the browser), but this is IMO a detail. It will still recreate the object to manipulate it (`read`/`write`).

Same apply for `getInMemoryCache` where it will fully re-create the cache on call.

Reply:
Interesting. That makes sense too.
I feel like by using `get` as a prefix, we imply this is a light operation. `create` is somewhat too heavy.

But do we want to guide users to do like the following? Or, is the current way `cache={getInfiniteHitsSessionStorageCache()}` is still fine?

```js
const cache = getInfiniteHitsSessionStorageCache();

function Hits() {
  return (
    <InfiniteHits
      hitComponent={Product}
      cache={cache}
    />
  );
}
```

What do you think?

Reply:
> I feel like by using get as a prefix, we imply this is a light operation. create is somewhat too heavy.

Well this is a very subjective feeling :wink: 

What I mean is that this `getX` function is still creating something new in all cases, not just getting something.

```js
getInfiniteHitsSessionStorageCache() === getInfiniteHitsSessionStorageCache() // false, for a `getX` function I would expect this to be true
```

My two cents, this is still a small naming question, not a big deal :slightly_smiling_face:

Reply:
Yes, I'd go with `create` too

Reply:
https://github.com/algolia/react-instantsearch/pull/2921/commits/d705bd900320fa861d6e2c7462f1aa77281eb7be